DEVELOPMENT.md

```diff
@@ -51,7 +51,7 @@ In order to use the local provider, a configuration block like the following may
 terraform {
   required_providers {
     pcluster = {
-      source  = "Terraform.local/local/pcluster"
+      source  = "terraform.local/local/pcluster"
```

Comment:
does this not map to a pcluster version?

Reply:
Clarified offline. No, terraform project will have a semantic versioning decouple from pcluster's.